MAUI/Calendar/customizations.md

```diff
@@ -648,4 +648,235 @@ this.calendar.View = CalendarView.Decade;
+        </DataTemplate>
+    </calendar:SfCalendar.SelectionCellTemplate>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+public partial class MainPage : ContentPage
+{
+    private DataTemplate template;
+    public MainPage()
+    {
+        InitializeComponent();
+        template = new DataTemplate(() =>
+        {
+            Grid grid = new Grid();
+            grid.BackgroundColor = Colors.Green;
+
+            Label label = new Label();
+            label.SetBinding(Label.TextProperty, "Date.Day");
+            label.HorizontalOptions = LayoutOptions.Center;
+            label.VerticalOptions = LayoutOptions.Center;
+            label.Padding = new Thickness(2);
+            grid.Children.Add(label);
+
+            return grid;
+        });
+        calendar.SelectionCellTemplate = template;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N>
+The `SelectionCellTemplate` is applicable only when the [SelectionMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html?tabs=tabid-6%2Ctabid-12%2Ctabid-18%2Ctabid-50%2Ctabid-10%2Ctabid-8%2Ctabid-14%2Ctabid-4%2Ctabid-22%2Ctabid-26%2Ctabid-24%2Ctabid-16%2Ctabid-2%2Ctabid-20#Syncfusion_Maui_Calendar_SfCalendar_SelectionMode) is set to Single. It is not applied in Multiple or Range selection modes. It's applicable or Month cell template also.
+
+## Selection cell appearance on month view using DataTemplateSelector
+
+You can customize the appearance of selected date cells in the SfCalendar's `MonthView` by using the [SelectionCellTemplate]() property within the `SfCalendar`. By combining it with a DataTemplateSelector, you can apply different styles for selected cells based on specific conditions.
+
+The DataTemplateSelector lets you define how selected cells should appear, depending on factors like the day of the week, the month, or other custom logic. This approach provides more flexibility in styling selected dates in the MonthView.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<Grid>
+    <Grid.Resources>
+        <DataTemplate x:Key="normalDateTemplate">
+            <Grid Background = "Pink" >
+                <Label HorizontalTextAlignment="Center" VerticalTextAlignment="Center" TextColor="Black" Text="{Binding Date.Day}"/>
+            </Grid>
+        </DataTemplate>
+        <DataTemplate x:Key="todayDateTemplate">
+            <Grid Background = "PaleGreen">
+                <Label HorizontalTextAlignment="Center" VerticalTextAlignment="Center" TextColor="Black" Text="Selection"/>
+            </Grid>
+        </DataTemplate>
+        <DataTemplate x:Key="leadingTraililngDateTemplate">
+            <Grid Background = "Purple">
+                <Label HorizontalTextAlignment="Center" VerticalTextAlignment="Center" TextColor="Black" Text="{Binding Date.Day}"/>
+            </Grid>
+        </DataTemplate>
+        <local:MonthCellTemplateSelector x:Key="monthCellTemplateSelector"
```

Comment:
MonthCellTemplateSelector

Here you added sample for Selection template- but mentioned as MonthCellTemplateSelector- Check